docs/LaunchChangelog.md

```diff
@@ -378,7 +378,7 @@ The S3 client in 2.0 is drastically different from the client in 1.11, because i
 | `doesBucketExistV2` | `headBucket` |
 | `doesObjectExist` | `headObject` |
 | `enableRequesterPays` | `putBucketRequestPayment` |
-| `generatePresignedUrl` | [Not Supported](https://github.com/aws/aws-sdk-java-v2/issues/849) |
+| `generatePresignedUrl` | [~~Not Supported~~](https://github.com/aws/aws-sdk-java-v2/issues/849) [S3Presigner](https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/examples-s3-presign.html) |
```

Comment:
Thanks for doing this! Should we remove the strike-through?

Reply:
I think the strike-through indicates work, that things have changed since the last time you saw this document.

Reply:
That's definitely beneficial. I wasn't sure if people read it multiple times for updates, and I figured the commit history exists if people want to know what's changed. I think it's one of those "fine either way" things.